src/js/components/CreateServiceModalServicePickerOption.js

```diff
@@ -0,0 +1,48 @@
+import {Hooks} from 'PluginSDK';
+import React from 'react';
+
+import CreateServiceModalServicePickerOptionContent from './CreateServiceModalServicePickerOptionContent';
+import CreateServiceModalServicePickerOptionImage from './CreateServiceModalServicePickerOptionImage';
+import Panel from './Panel';
+
+function getChildOfType(children, type) {
+  return React.Children.toArray(children).filter(
+    (child) => child.type === type
+  );
+}
+
+function CreateServiceModalServicePickerOption({children, onOptionSelect}) {
+  const columnClass = Hooks.applyFilter(
```

Comment:
The option needs classes that define its width; if it's a 3x1 grid, it needs `column-small-4` to consume 33% of its parent's width. If it's a 2x2 grid, it needs `column-small-6` to consume 50%.